incubator/hydra/templates/NOTES.txt

```diff
@@ -0,0 +1,28 @@
+{{- if and (not .Values.postgresql.enabled) (not .Values.config.hydra.databaseURL) }}
+#########################################################################
+## WARNING: You have not specified a Database to connect to.           ##
+#########################################################################
+
+This deployment will be incomplete until you provide the databaseURL to connect to, or enable the postgresql dependency
+{{- end }}
+
+To connect to Hydra, you can use:
+
+{{- if contains "NodePort" .Values.service.type }}
+  export NODE_IP=$(kubectl get nodes --namespace {{ .Release.Namespace }} -o jsonpath="{.items[0].status.addresses[0].address}")
+  echo http://$NODE_IP/
+{{- else if contains "LoadBalancer" .Values.service.type }}
+  export SERVICE_IP=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ template "hydra.fullname" . }} -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
+  echo http://$SERVICE_IP/
```

Comment:
I think the idea here is that it's proxied to :80, since it's a load balancer, no?